dot_config/opencode/kb-collectors/openspec.md

```
@@ -0,0 +1,44 @@
+For each date being enriched, read every `$KB_ROOT/openspec/*/changes/archive/<date>-*/kb-meta.yaml`. Collect the `worktree:` value from each. That set is the exclusion list passed to the `opencode` collector.
+
+```bash
+# Collect worktrees for a given date
+for meta in $KB_ROOT/openspec/*/changes/archive/<date>-*/kb-meta.yaml; do
+  grep '^worktree:' "$meta" | awk '{print $2}'
+done
+```
+
+Then read each archived change's `design.md`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)